azkaban-common/src/main/java/azkaban/imagemgmt/daos/ImageVersionDaoImpl.java

```diff
@@ -412,4 +432,18 @@ public void updateImageVersion(final ImageVersion imageVersionRequest)
       return imageVersions;
     }
   }
+
+  /**
+   * ResultSetHandler implementation class for fetching image version
+   */
+  public static class CheckInvalidImageVersionHandler implements ResultSetHandler<Boolean> {
+
+    @Override
+    public Boolean handle(final ResultSet rs) throws SQLException {
+      if (!rs.next()) {
+        return true;
+      }
+      return rs.getBoolean("is_invalid");
+    }
+  }
 }
```

Comment:
Nit: Please apply check style and save actions.

Reply:
Check style is already applied.

Reply:
New line in the end.

Reply:
Done